src/mem/sizeclass.h

```diff
@@ -4,35 +4,31 @@
 
 namespace snmalloc
 {
-  constexpr static uint16_t get_initial_bumpptr(uint8_t sc, bool is_short);
-  constexpr static uint16_t get_initial_link(uint8_t sc, bool is_short);
-  constexpr static size_t sizeclass_to_size(uint8_t sizeclass);
-  constexpr static size_t sizeclass_to_cache_friendly_mask(uint8_t sizeclass);
-  constexpr static size_t sizeclass_to_inverse_cache_friendly_mask(uint8_t sc);
-  constexpr static uint16_t medium_slab_free(uint8_t sizeclass);
-
-  static inline uint8_t size_to_sizeclass(size_t size)
-  {
-    // Don't use sizeclasses that are not a multiple of the alignment.
-    // For example, 24 byte allocations can be
-    // problematic for some data due to alignment issues.
-    return static_cast<uint8_t>(
-      bits::to_exp_mant<INTERMEDIATE_BITS, MIN_ALLOC_BITS>(size));
-  }
+  // Both usings should compile
+  using sizeclass_t = size_t;
+  //  using sizeclass_t = uint8_t;
+
+  constexpr static uint16_t get_initial_offset(sizeclass_t sc, bool is_short);
+  constexpr static size_t sizeclass_to_size(sizeclass_t sizeclass);
+  constexpr static size_t sizeclass_to_cache_friendly_mask(sizeclass_t sizeclass);
+  constexpr static size_t sizeclass_to_inverse_cache_friendly_mask(sizeclass_t sc);
+  constexpr static uint16_t medium_slab_free(sizeclass_t sizeclass);
+  static sizeclass_t size_to_sizeclass(size_t size);
 
-  constexpr static inline uint8_t size_to_sizeclass_const(size_t size)
+
+  constexpr static inline sizeclass_t size_to_sizeclass_const(size_t size)
   {
     // Don't use sizeclasses that are not a multiple of the alignment.
     // For example, 24 byte allocations can be
     // problematic for some data due to alignment issues.
-    return static_cast<uint8_t>(
+    return static_cast<sizeclass_t>(
       bits::to_exp_mant_const<INTERMEDIATE_BITS, MIN_ALLOC_BITS>(size));
   }
 
   constexpr static inline size_t large_sizeclass_to_size(uint8_t large_class)
   {
     return bits::one_at_bit(large_class + SUPERSLAB_BITS);
-  }
+  } 
```

Comment:
Space.

Reply:
Fix clangformat to actually run on builds, so this should be fixed if it passes CI now.